sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/SyncAccountRepository.kt

```diff
@@ -882,11 +901,11 @@ enum class AccountErrorCodes(val code: Int) {
     EXCHANGE_FAILED(56),
 }
 
-enum class CodeType {
-    RECOVERY,
-    CONNECT,
-    EXCHANGE,
-    UNKNOWN,
+sealed interface CodeType {
+    data class Recovery(val code: RecoveryCode) : CodeType
```

Comment:
since we have different code formats, do you think we should put here b64Code for clarity?

Reply:
Done